tests/bson/bson-toPHP-003.phpt

```diff
@@ -9,6 +9,7 @@ class MyClass
 {
 }
 
+#[\AllowDynamicProperties]
```

Comment:
I suppose there's no way to add this attribute to internally defined classes, right? If so, we could use this for testing preservation of dynamic properties during object cloning in lieu of restricting the test to pre-8.2.

Reply:
It can be added to internally defined classes using `zend_add_class_attribute`, although I'd much rather take advantage of this new functionality and disallow dynamic attributes on our BSON classes.